### PR TITLE
Display pokemons based on real images

### PR DESCRIPTION
**Pull request by LinkOut**
- Using original images as markers
- Removed some useless and annoying Toasts Issue #113
- Updated user position marker because was too big the original one (using a pokeball now)
- Added Glide lib to fetch images

Preview:

### CODE REVIEW

File: app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java [hunk 1]
```diff
@@ -164,24 +169,45 @@ private void setPokemonMarkers(final List<CatchablePokemon> pokeList){
...
+                            }
+                        });
             }
         } else {
             Toast.makeText(getContext(), "The map is not initialized.", Toast.LENGTH_LONG).show();
         }
     }
 
+    //Getting correct pokemon Id eg: 1 must be 001, 10 must be 010
+    private String getCorrectPokemonIdNumber (int pokemonId){
```
Comment (s7092910) [inline app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java:200]: I might call it getPokemonImageId or something along that line.

**Comment by s7092910** [comment]
Looks good to me

File: app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java (hunk 1 above)
Comment (0legg) [inline app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java:200]: I'd replace this unstable manual converter with usage of `String.format("%03d", pokemonId)` for now

**Comment by 0legg** [comment]
Makes a little sense for me, because now we're introducing additional dependency and source of errors instead of having all the images bundled inside the app — it's much easier to have it all once instead of fetching from network and dealing with possible network/cache issues.

**Comment by s7092910** [comment]
@0legg Legal Reasons. The pokemon images are copyrighted, legally we can't use them, but legally we are allow to hyperlink to them from a site that is hosting them even if they aren't legally allowed to use them. In the EU of course

**Comment by 0legg** [comment]
@s7092910 Now it make more sense, but app itself is still vulnerable, cause removing image from 3rd-party site leads to app breaking. So there is no silver bullet, unless you are Nintendo itself. And if you're Nintendo, you'd better sell LED wristband for $30

**Comment by omkarmoghe** [comment]
lgtm, can you fix the conflicts real quick?

**Comment by LinkOut** [comment]
Closed for now, waiting for all those PR's LeLz dont wanna fix conflicts all times

**Pull request closed without merging**